### ISSUE TITLE
Use :lang() and a Sass map to deal with translations of custom file input text?

### ISSUE THREAD

**Issue by cvrebert** [twbs/bootstrap#18922]
See https://developer.mozilla.org/en-US/docs/Web/CSS/:lang

Something along the lines of:
```scss
$custom-file-text: (
  button-label: (
    en: "Browse"
  ),
  placeholder: (
    en: "Choose file..."
  )
) !default;

@each $lang, $text in map-get($custom-file-text, button-label) {
  ...:lang($lang) {
    content: $text;
  }
}
```

Folks using other languages can then set new entries in the map via their custom variables file to add their translations.

CC: @mdo for feedback

**Comment by mdo** [comment]
Oh, neat. That'd be cool to implement I think. Would we do it anywhere else?

**Comment by cvrebert** [comment]
We don't have language-specific `content` anywhere else in the CSS besides the custom file inputs, so no.

**Comment by cvrebert** [comment]
Implemented in #18925.
